src/main/scala/is/hail/methods/VariantQC.scala

```diff
@@ -129,7 +129,10 @@ object VariantQC {
     "rExpectedHetFrequency" -> TFloat64(),
     "pHWE" -> TFloat64())
 
-  def apply(vsm: VariantSampleMatrix, root: String): VariantSampleMatrix = {
+  def apply(vsm: VariantSampleMatrix, root: String = "va.qc"): VariantSampleMatrix = {
+    require(vsm.wasSplit)
```

Comment:
wasSplit is going away, right? I kinda think the right thing to do is to never check splittedness anywhere, but to error on gt>2 on methods that don't support it.

Reply:
Yes, I sure hope so.  That's a good idea, and simple.  My idea was to have fixed length (length 2) array for the alleles: http://dev.hail.is/t/the-future-of-types/20, point 4.